app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+* Continuing to support the existing experimentation (Normandy) workflow
+
+## Considered Options
+
+* Build On Top Of Experimenter
+* Start A New Application In Python/Django
+* Start A New Application in TypeScript/Node
+
+## Decision Outcome
+
+Chosen option: TBD
+
+### Positive Consequences
+
+* [e.g., improvement of quality attribute satisfaction, follow-up decisions required, …]
+* …
+
+### Negative Consequences
+
+* [e.g., compromising quality attribute, follow-up decisions required, …]
+* …
+
+## Pros and Cons of the Options
+
+### Build On Top Of Experimenter
+
+Much of the existing Experimenter infrastructure can be reused, which gives Project Nimbus a well established framework to build on top of.
+
+- Pros
+  - Single source of truth for all past/future experiments
+    - Easily discover past results
+    - Correlate past/future experiments easily
+    - Generate high level reports for leadership/stakeholders across all experiment types
+    - Consistent APIs for dependent automated tasks (ie data engineering/science)
+  - Reuse existing components
+    - Database schemas are largely reusable
+    - Public APIs work seamlessly for Normandy and Nimbus experiments
+    - External service integrations (bugzilla/remote settings/email/auth0) work seamlessly
+  - Comprehensive development/testing/deployment infrastructure in place
+    - 100% unit testing coverage across Python/JS/TS
+    - Automated Selenium Integration tests for frontend
+    - Continuous Deployment
+- Cons
+  - Existing workflow/UI not applicable
+  - Some database schemas require refactoring to isolate Normandy and Nimbus concerns
+  - Careful consideration that adding Nimbus features does not disturb existing functionality
+  - Lack of static typing (mypy can be added with effort)
+  - Supporting both Normandy and Nimbus over time could cause unwanted complexity/duplication
+  - [Current Experimenter codebase may not align with FXA tools/processes/practices](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+  - Somewhat constricted by core designs decisions that were made in service of existing Normandy experimentation workflows that may not apply to Nimbus
+
+### Start A New Application In Python/Django
+
+Creating a new web application in Python/Django would allow us to reuse much of the existing code/infrastructure where applicable while clearly isolating the concerns of supporting existing experiment types and building out new Nimbus functionality.
+
+- Pros
+  - Start by cloning the repo and deleting all the unnecessary UI/DB/APIs
+  - Get a big head start by not needing to reimplement the reusable parts
+  - Isolate Nimbus concerns without fear of breaking Normandy experiments
+  - Consistent development and deployment workflow for both old and new applications
+  - Opportunity to rethink core code/architectural/tooling decisions where appropriate
+- Cons
+  - Lose single source of truth
```

Comment:
I mention this in another comment, but to reiterate/expand here: how much weight does this con have? If we want to check various prior experiments often even after they aren't supported any longer, it seems having access to a history of Normandy experiments would be handy to query in Nimbus?

Reply:
I think @shell1 addresses that well here:

https://github.com/mozilla/experimenter/pull/3259#discussion_r475719606